apps/governance/src/routes/proposals/proposal/Proposal.graphql

```diff
@@ -162,90 +160,139 @@ query Proposal($proposalId: ID!) {
-                                key
+                      ... on DataSourceDefinitionExternal {
+                        sourceType {
+                          ... on DataSourceSpecConfiguration {
+                            signers {
+                              signer {
+                                ... on PubKey {
+                                  key
+                                }
+                                ... on ETHAddress {
+                                  address
+                                }
+                              }
+                            }
+                            filters {
+                              key {
+                                name
+                                type
                               }
-                              ... on ETHAddress {
-                                address
+                              conditions {
+                                operator
+                                value
                               }
                             }
                           }
-                          filters {
-                            key {
-                              name
-                              type
-                            }
+                        }
+                      }
+                    }
+                  }
+                  # dataSourceSpecForTradingTermination {
+                  #   sourceType {
+                  #     ... on DataSourceDefinitionInternal {
+                  #       sourceType {
+                  #         ... on DataSourceSpecConfigurationTime {
+                  #           conditions {
+                  #             operator
+                  #             value
+                  #           }
+                  #         }
+                  #       }
+                  #     }
+                  #     ... on DataSourceDefinitionExternal {
+                  #       sourceType {
+                  #         ... on DataSourceSpecConfiguration {
+                  #           signers {
+                  #             signer {
+                  #               ... on PubKey {
+                  #                 key
+                  #               }
+                  #               ... on ETHAddress {
+                  #                 address
+                  #               }
+                  #             }
+                  #           }
+                  #           filters {
+                  #             key {
+                  #               name
+                  #               type
+                  #             }
+                  #             conditions {
+                  #               operator
+                  #               value
+                  #             }
+                  #           }
+                  #         }
+                  #       }
+                  #     }
+                  #   }
+                  # }
```

Comment:
Not an issue, but did you want to leave this commented block in the PR?

Reply:
wanted to double-check with @sam-keen but I think this part on proposals was commented some time ago, when  we couldn't query for internal data source specs. 

I think we're doing a REST call for that internal data source specs for proposals - I think we should rather handle that in another PR